docs/install_guides/configurations-for-intel-npu.md

```diff
@@ -22,31 +22,68 @@ Linux
 Prerequisites
 ++++++++++++++++++++
 
-Ensure that make, gcc, and Linux kernel headers are installed. Use the following command to install the required software:
+* Supported operating systems: Ubuntu 20.04 / 22.04 LTS
+
+* Ensure that make, gcc, and Linux kernel headers are installed. Use the following command to install the required software:
 
 .. code-block:: sh
 
    sudo apt-get install gcc make linux-headers-generic
 
 
-Configuration steps
+Manual Configuration steps
 ++++++++++++++++++++
 
+(i) After downloading the latest driver, you can unpack it with below command. Please substitute the relevant release version number:
+
+    `tar -xf vpu-linux-drivers-ubuntu2204-release-<version number>.tar.gz`
```

Comment:
where can I download it?

Reply:
I don't have the location where the final revision of the driver will be available from. Possibly from the same location as Openvino package?